lms/djangoapps/commerce/api/v0/views.py

```diff
@@ -140,7 +140,9 @@ def post(self, request, *args, **kwargs):
                 msg = Messages.NO_SKU_ENROLLED.format(
                     enrollment_mode=default_enrollment_mode.slug,
                     course_id=course_id,
-                    username=user.username
+                    course_name=course.display_name,
```

Comment:
Is it possible for `course` to be `None` here?

Reply:
Good point. I'm not sure, so I'll update this to account for that.